DigitalOffice/Services/TimeManagementService/Validators/CreateLeaveTimeRequestValidator.cs

```diff
@@ -0,0 +1,55 @@
+﻿using FluentValidation;
+using LT.DigitalOffice.TimeManagementService.Models;
+using LT.DigitalOffice.TimeManagementService.Repositories.Interfaces;
+using Microsoft.AspNetCore.Mvc;
+using System;
+
+namespace LT.DigitalOffice.TimeManagementService.Validators
+{
+    public class CreateLeaveTimeRequestValidator : AbstractValidator<CreateLeaveTimeRequest>
+    {
+        private readonly ILeaveTimeRepository repository;
```

Comment:
Field 'repository' is assigned but its value is never used

Reply:
Fixed.